### PR TITLE
fixed metamask bug 

### PR DESCRIPTION
**Pull request by afolabiopakunle**

### CODE REVIEW

File: src/app/pages/asset-details/asset-details.component.html [hunk 1]
```diff
@@ -223,34 +226,65 @@ <h5 class="heading-h3 bid-head" >{{auction.sellNowPrice}} ONE</h5>
...
                                                                type="text" name="amount" [(ngModel)]="amount" placeholder="Current Bid: {{auction.bids[0]['bid']}} ONE ({{this.auctionValue | currency}}) "  type="number" id="amount" name="amount"required>
                                                     </div>
                                                 </div>
 
-                                                <div class="action-otr" *ngIf="visible && accountFound">
-                                                    <a (click)="bid()" class="btn-fill btn-bid">Place a Bid</a>
+                                                <div class="action-otr" style="flex: 1" *ngIf="visible && accountFound">
+                                                    <a (click)="bid()" class="btn-fill btn-bid" style="border-top-left-radius: 0; border-bottom-left-radius: 0;">Place a Bid</a>
+                                                </div>
+                                               </div>
```
Comment (spockula) [inline src/app/pages/asset-details/asset-details.component.html:256]: Please comment out stripe implementation as we will be pushing the develop build to live, once the setup is complete for stripe we can add it to the live build

**Review by spockula — CHANGES_REQUESTED** [review]
Please check the comment on asset details html and make changes, then push again. Well done by the way.

File: src/app/pages/asset-details/asset-details.component.html (hunk 1 above)
Comment (afolabiopakunle) [reply src/app/pages/asset-details/asset-details.component.html:257]: Requested updates implemented as the stripe codes have been commented out

**Review by spockula — APPROVED** [review]
well done!

**Pull request merged**